### PR TITLE
Add css url filter and Change enqueue handle

### PR DESCRIPTION
**Pull request by munyagu**
I added a filter so that users can change to custom css.
And change enqueue handle for css.
'dark_mode_css' appears as 'dark_mode_css - css'

### CODE REVIEW

File: dark-mode.php [hunk 1]
```diff
@@ -99,10 +99,20 @@ public static function load_dark_mode_css() {
 			do_action('doing_dark_mode');
 
 			// Register the Dark Mode stylesheet
-			wp_register_style('dark_mode_css', plugins_url('dark-mode', 'dark-mode') . '/darkmode.css', array(), '1.0');
+			/**
+			 * Filters the Dark Mode stylesheet.
+			 *
+			 * Prevents image captions from being appended to image HTML when inserted into the editor.
```
Comment (danieltj27) [inline dark-mode.php]: Remove this line - looks like it's from some other function documentation.

File: dark-mode.php [hunk 2]
```diff
@@ -99,10 +99,20 @@ public static function load_dark_mode_css() {
...
+			 *
+			 * @since
```
Comment (danieltj27) [inline dark-mode.php]: Please add 1.1 to the since version.

File: dark-mode.php [hunk 3]
```diff
@@ -99,10 +99,20 @@ public static function load_dark_mode_css() {
...
+			 *
+			 * @param string URL of default CSS for dark mode.
```
Comment (danieltj27) [inline dark-mode.php]: Add the name of the name of the variable and capitalise the name Dark Mode, so it'd appear as `@param string $css_url URL of default CSS for Dark Mode.`

**Comment by munyagu** [comment]
Thank you for your review.
I fixed documentation.
Since I am not used to Github operation, please let me know if the correction method is wrong.

**Pull request merged**